Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -454,18 +458,21 @@ private class TopSitesDataSource: NSObject, UICollectionViewDataSource {
     private let BackgroundFadeInDuration: NSTimeInterval = 0.3
 
     init(profile: Profile, data: Cursor<Site>) {
-        self.data = data
         self.profile = profile
+        self.topSitesCursor = TopSitesCursor(profile: profile, child1: data.asArray(), child2: SuggestedSites.asArray())
```

Comment:
If we're going to name the wrapper cursor class `TopSitesCursor` then we can name the children `suggestedSites/historySites` instead. An alternative would be to make this a generic 'wrapper' cursor and use a data structure to infer positioning like an array to store children.